src/main/java/uk/gov/hmcts/reform/sscs/controller/SyaController.java

```diff
@@ -35,16 +36,21 @@
 import uk.gov.hmcts.reform.sscs.model.draft.Draft;
 import uk.gov.hmcts.reform.sscs.model.draft.SessionDraft;
 import uk.gov.hmcts.reform.sscs.service.SubmitAppealService;
+import uk.gov.hmcts.reform.sscs.service.SubmitAppealServiceV2;
 
 @RestController
 @Slf4j
 public class SyaController {
 
     private final SubmitAppealService submitAppealService;
+    private final SubmitAppealServiceV2 submitAppealServiceV2;
+    @Value("${feature.submit-appeal-service-v2.enabled}")
+    private boolean isSubmitAppealV2Enabled;
 
     @Autowired
-    SyaController(SubmitAppealService submitAppealService) {
+    SyaController(SubmitAppealService submitAppealService, SubmitAppealServiceV2 submitAppealServiceV2) {
```

Comment:
I think it would be better to use an interface for these V1 & V2 SubmitAppealService

Reply:
Replaced it with SubmitAppealServiceBase